server/utils/noIndexMiddleware.test.js

```diff
@@ -0,0 +1,53 @@
+const httpMocks = require('node-mocks-http')
+
+const noIndexMiddleware = require('./noIndexMiddleware')
+
+const mockedServerHost = 'mockedserverhost.example.com'
+jest.mock('../configuration', () => ({
+  server: { hostUrl: 'https://mockedserverhost.example.com' },
+}))
+
+describe('noIndexMiddleware', () => {
+  it('should not set "X-Robots-Tag" header if "X-Forwarded-Host" is missing', () => {
+    const next = jest.fn()
+    const { req, res } = httpMocks.createMocks({
+      headers: {},
+    })
+
+    noIndexMiddleware(req, res, next)
+
+    expect(res.getHeader('X-Robots-Tag')).toBeUndefined()
+    // expect(next).toHaveBeenCalledOnce()
+    expect(next).toHaveBeenCalledTimes(1)
```

Comment:
Same comment as I gave @amirhossein-haerian in the other PR here: https://github.com/KTH/kursinfo-web/pull/357/files#r1650788842
But in this case, please either include `jest-extended`, or remove the commented-out lines :)